util/dashboard/fp_004.py

```diff
@@ -29,13 +29,13 @@
 import re
 from lib import url_exists
 
-import requests
 
-# regex pattern to match dated version IRIs
-pat = r'http:\/\/purl\.obolibrary\.org/obo/.*/([0-9]{4}-[0-9]{2}-[0-9]{2})/.*'
+# regex pattern to match purl obolibrary url
+pat = r'http:\/\/purl\.obolibrary\.org/obo/.*/.*/.*'
 PATTERN = re.compile(pat)
 #: Official regex for semantic versions from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
-SEMVER_PATTERN = re.compile(r"^(0|[1-9]\d*)\.(0|[1-9]\d*)(\.(0|[1-9]\d*))?(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$")
+# Simplify semantic versions to accept only numbers with or without point, .eg 1.1 or 11
+SEMVER_PATTERN = re.compile(r"^(0|[1-9]\d*)(\.)?(0|[1-9]\d*)((\.)?(0|[1-9]\d*))?$")
```

Comment:
```suggestion
SEMVER_PATTERN = re.compile(r"^(0|[1-9]\d*)(\.)?(0|[1-9]\d*)?((\.)?(0|[1-9]\d*))?$")
```

I think this needs a tweak to allow single digit numbers, rather than require two digits.